Unity quest content description (#166)
vrchat-community/creator-docs: Docs/docs/platforms/android/setting-up-unity-for-creating-quest-content.md

updatedAt: "2019-10-28T19:23:09.636Z"
sidebar_position: -1
---
![Building for Mobile instructions](/img/setting-up-unity-for-creating-quest-content-1ac8b19-VRChat_QuestContent_QuickStart.png)

If you're starting a brand new project, this won't take long at all. However, if you're converting a Windows platform project to an Android platform project, you will have to convert your assets appropriately. This can take quite a while for larger projects.

For more details on best practices when working with dual-platform projects, read our documentation on [Cross-Platform Setup](/platforms/android/cross-platform-setup).

## 3 Steps to get your world on VRChat for Quest

> ### 1. Open Your Build Settings
> 
> :::tip 
> Pros use the hotkeys Ctrl + Shift + B.
> :::

Reviewer: Should this contain a link to Unity's documentation?

The shortcut is helpful, but admonitions should only be used for information that's very important.
```suggestion
### 1. Open Your Build Settings

You can access your [Build Settings](https://docs.unity3d.com/Manual/BuildSettings.html) from Unity's main menu by going to "File" -> "Build Settings". Or you can use the keyboard shortcut `Ctrl` + `Shift` + `B`.
```

> ### 2. Switch Platform to Android
> 
> :::caution Requires Additional Setup
> If the Android option isn't appearing, [you'll need to install Unity's Android SDK.](https://docs.unity3d.com/Manual/android-sdksetup.html)
> 
> :::
> 
> Click "Android" -> "Switch Platform". Unity will automatically convert assets for Android-ready, it will take a while for larger projects with lots of assets.
> 

Reviewer: ```suggestion
### 2. Switch Platform to Android

:::caution Requires Additional Setup
If the Android option isn't appearing, [you need to install Unity's Android SDK.](https://docs.unity3d.com/Manual/android-sdksetup.html)

:::

Click "Android" -> "Switch Platform". Unity will reimport your assets for Android. This takes a while for large projects with many assets.

```

Author: Added in https://github.com/vrchat-community/creator-docs/pull/166/commits/2b0d4b73b4355c39c4e7b9e9c6aea9df6709d511

### 3. Publish -> New Build

> That's it! Your content is ready to check out on the Quest! Note that **you must publish**. Testing locally is not supported for Quest.

Reviewer: Users need to upload their content, but they don't need to publish it.
```suggestion
That's it! Your content is now available on Quest! Note that you must **upload** your content to test it. You can't use the SDK's "Build and Test" feature for Android content.
```
